build.sbt

```diff
@@ -60,6 +60,7 @@ libraryDependencies ++= Seq(
   "com.typesafe.akka" %% "akka-http" % akkaHttpVersion,
   "com.typesafe.akka" %% "akka-parsing" % akkaHttpVersion,
   "com.typesafe.akka" %% "akka-stream" % akkaVersion,
+  "com.typesafe.akka" %% "akka-slf4j" % akkaVersion,
```

Comment:
Is the new dependency needed for purposes of this PR ?

Reply:
And the configuration section I added should be always present when SLF4J